style.css

```diff
@@ -244,11 +244,19 @@
 
   .Contacting .btn {
     display: block;
+    margin-left: 3%;
```

Comment:
- Hey, great making the change, however, your design is still not close. to fix this, increase the width on your inputs and textarea to 95% and remove this added padding.